packs/device42/actions/device_name_list.py

```diff
@@ -0,0 +1,66 @@
+from st2actions.runners.pythonrunner import Action
+import requests
+
+
+class DeviceNameList(Action):
+    def run(self, type=None, service_level=None, in_service=None, customer=None, tags=None,
```

Comment:
Hm, that's quite a lot of arguments / parameters...

Do they change often? If so, it would be good to automate the code and metadata generation if possible. If not, it's OK the way it is :)